docs/044-ovos_installer.md

```diff
@@ -0,0 +1,13 @@
+# OVOS Installer
+
+[OVOS-Installer](https://github.com/OpenVoiceOS/ovos-installer) is the newest way to install OVOS and is probably the easiest way to install on an existing system.
+
+You do need a few packages installed on your host system before starting.
```

Comment:
Speed-readers and document parsers are looking for this section to be subtitled "Prerequisites"